test/e2e/webdriver/driver.js

```diff
@@ -534,12 +535,12 @@ class Driver {
     return await this.driver.getAllWindowHandles();
   }
 
-  async waitUntilXWindowHandles(x, delayStep = 1000, timeout = this.timeout) {
```

Comment:
Under MV3, there will always be 1 more window handle than previously assumed, so this is a global fix for `waitUntilXWindowHandles`, to accomodate the MV3 offscreen document